src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.list.FunnyList;
+
+/**
+ * Main class for the Duke application.
+ * This class initializes and manages the Duke application. It handles user interaction,
+ * task processing, and exception handling.
+ */
+public class Duke {
+    private Storage storage;
+    private FunnyList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs a Duke instance with the specified file path.
+     *
+     * @param filePath The file path to the text file to load and save task data.
+     */
+    public Duke(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.tasks = new FunnyList(storage.load());
+        } catch (DukeException e) {
+            ui.showLoadingError();
+            this.tasks = new FunnyList();
+        }
+    }
+
+    /**
+     * Main method to start the Duke application.
```

Comment:
I am unsure whether this Javadoc comment should start with a verb instead. Maybe something like `Starts the Duke application`?

Reply:
Hi thanks for pointing that out! Have resolved it!